packages/host/app/components/operator-mode/code-submode/playground/playground-content.gts

```diff
@@ -457,15 +480,26 @@ export default class PlaygroundContent extends Component<Signature> {
 
   private createNewCard = task(async () => {
```

Comment:
does this need to be restartable? what happens if the user selects this again very quickly before the async finishes?

Reply:
Yes, missed this during a recent change